docs/adguard-vpn-for-windows/installation.md

```diff
@@ -22,3 +22,29 @@ If you decide to remove AdGuard VPN from your computer, use one of three options
 2. Click *Start* → *Settings* → *Apps* → *Applications and features*. Find AdGuard VPN in the list, click it and choose *Uninstall*.
 
 3. Open the *Control Panel*, then click *Programs* → *Programs and Features* → *Uninstall a program*. Find AdGuard VPN in the list, right-click it and select *Uninstall*.
+
+### Advanced uninstallation {#advanced}
+
+In case regular uninstall doesn't work for any reason, you can try to use an advanced method. First of all, you need to [download the uninstaller tool](https://cdn.adtidy.org/public/Adguard/tools/Uninstall_Utility.zip) created by our developers. Extract the archive to any folder on your PC and run the **Adguard.UninstallUtility.exe** file, and allow the app to make changes to your device. Then follow the instruction below:
+
+* Choose *Standard uninstall*, ***Delete AdGuard VPN*** and click *Uninstall*.
+
+![Standard uninstall *border](https://cdn.adtidy.org/content/kb/ad_blocker/windows/installation/standard-uninstall.png)
+                                 
+* Wait until uninstall is finished — there will be a string in the window:
+`[OK] Uninstall finished`
+                            
+![Uninstall finished *border](https://cdn.adtidy.org/content/kb/ad_blocker/windows/installation/standard-uninstall-2.png)
+
+> Follow the next steps only if after performing first two steps wasn’t enough for some reason. Usually it can be determined by seeing incorrect reinstall of AdGuard, but we strongly suggest contacting our support before using steps 3-4 of advanced uninstall instruction.
+
+* Choose *Advanced uninstall*, ***Delete AdGuard VPN*** and click *Uninstall*.
+
+![Advanced uninstall *border](https://cdn.adtidy.org/content/kb/ad_blocker/windows/installation/advanced-uninstall.png)
+                                 
+* Wait until uninstall is finished — there will be a string in the window:
```

Comment:
same as above

Reply:
To emphasize that you need to choose AdGuard VPN (check the screenshot)